### ISSUE TITLE
Selection in coastal Synechococcus (cyanobacteria) populations evaluated from environmental metagenomes.

### ISSUE THREAD

**Issue by pschloss** [SchlossLab/march_madness_2015#52]
Selection in coastal Synechococcus (cyanobacteria) populations evaluated from environmental metagenomes.
Tai V, Poon AF, Paulsen IT, Palenik B.
PLoS One. 2011;6(9):e24249. doi: 10.1371/journal.pone.0024249. Epub 2011 Sep 9.
http://www.ncbi.nlm.nih.gov/pubmed/21931665

**Comment by pschloss** [comment]
The paper by Tai and colleagues describes the development and implementation of methods to calculate the dN/dS ratios for genes retrieved from shotgun metagenomic sequence datasets. Here they tested the method using an enriched Synechococcus metagenome that was obtained by initially using flow cytometry. As we know, one of the key problems with metagenomics is problem of assembling genomes and being confident that a set of co-assembled reads came from the same individual. Here they used 454 data and mapped the reads to publicly available Synechococcus genome sequences. As they used 454, I suspect they had limited sequence read depth to assemble the reads. My understandign from talking with Titus Brown and others is that if a de novo assembler is used you can be confident that the reads came from the same population but not the same individual. Yet, when we map reads to a genome, there's room for all sorts of crazy stuff to happen so that it's not clear at all whether you merely have a similar gene or the same host organism. Regardless, they calculated dN/dS ratios by mapping reads to genomes and identifying polymorphisms. For me, the paper really stalls after the description of the method. They find a bunch of crap under purifying and positive selection. One notable result was the identification of a ribosomal protein under positive selection, which is odd because you'd expect it to be under pretty heavy purifying selection since it's one of the most housekeeping of housekeeping genes. As you'd also expect, a lot of genes of unknown function were under positive selection as well. I'd love to be able to push this with human metagenomic data to see whether we see signs of positive/neutral/negative selection. We can do it and we have done it, but in the end the question is so what? what can we do with the results other than report a long list of genes? If we can get over that hurdle, then I think we will have a stronger paper than that of Tai and colleagues.

Overall: 6
Importance: 5
Innovation: 9
Who should read: @kdiverson @agelmore 
Tags: metagenomics, evolution, methods

**Comment by kdiverson** [comment]
@pschloss what about tying dn/ds in with some of the diet stuff @agelmore did? Or really anything that has an experimental component to it. We'll still end up with a list of genes but at least we could attach some meta information to them. That DeLong paper kind of did this with observational data at ALOHA. Baby steps.

**Comment by pschloss** [comment]
yeah i think if there was an experiment that would be good. but for dNdS wouldn't there have to be some type of mutation and selection? perhaps it wouldn't be selection for an allele of a specific gene but selection of a bug with a specific allele?
